src/components/App/App.tsx

```diff
@@ -5,36 +5,36 @@ import { getParsedCommandLineOfConfigFile, isThisTypeNode } from 'typescript';
 import { render } from '@testing-library/react';
 import Questions from '../Questions/Questions'
 
-interface QuestionsData {
-  questions: Array<QuestionData>
+export interface QuestionsType {
+  questionsProp: QuestionDataType[]
 }
 
-interface QuestionData {
+export interface QuestionDataType {
```

Comment:
Conventions for naming this would probably be `Question` or `QuestionData` - no need for the `Type` to be added on to the name.